src/send/EnterAmount.tsx

```diff
@@ -67,12 +73,38 @@ interface Props {
   onPressProceed(args: ProceedArgs): void
   disableProceed?: boolean
   children?: React.ReactNode
+  CTA: ComponentType<CTAProps>
```

Comment:
naming nit: maybe match with proceed to be consistent with other args? e.g., `ProceedComponent` ? Or rename other args to use Cta to be consistent.

Also, this needs to start with an upper case letter to render as a jsx element right?